packages/xdl/src/EmbeddedAssets.ts

```diff
@@ -327,6 +362,15 @@ async function _maybeConfigureExpoUpdatesEmbeddedAssetsAsync(config: EmbeddedAss
       androidManifestXmlPath
     );
   }
+
+  if (isLikelyFirstPublish) {
+    logger.global.warn(
+      '🚀 It looks like this your first publish for this project! ' +
+        "We've automatically set some configuration values in Expo.plist and AndroidManifest.xml. " +
```

Comment:
we might want to point to the full paths to Expo.plist and AndroidManifest.xml here so people can easily see if the path is incorrect (has sometimes happened when people rename their projects on ios) and open it with just cmd+clicking it